google/cloud/bigtable/internal/table_admin.cc

```diff
@@ -235,16 +235,15 @@ void TableAdmin::DeleteSnapshot(bigtable::ClusterId const& cluster_id,
       &AdminClient::DeleteSnapshot, request, "DeleteSnapshot", status);
 }
 
-void TableAdmin::ListSnapshotsImpl(
-    bigtable::ClusterId const& cluster_id,
-    std::function<void(google::bigtable::admin::v2::Snapshot)> const& inserter,
-    grpc::Status& status) {
+std::vector<google::bigtable::admin::v2::Snapshot> TableAdmin::ListSnapshots(
+    grpc::Status& status, bigtable::ClusterId const& cluster_id) {
```

Comment:
out-params (and in-out params) are typically listed last in the function signature, according to our style guide. So perhaps move `status` to be the last parameter.